attributes/default.rb

```diff
@@ -1,40 +1,74 @@
 
+# Organization name that is displayed in the UI header.
 node.default['ice']['company_name']                         = nil
 
+# Enables the ice processor.
 node.default['ice']['processor']['enabled']                 = true
-node.default['ice']['processor']['local_dir']                = '/var/ice_processor'
+
+# Local work directory for the Ice processor.
+node.default['ice']['processor']['local_dir']               = '/var/ice_processor'
+
+# Optional. Work around https://github.com/Netflix/ice/issues/100 by pre-creating
+# the directories it expects to find. Default: false
 node.default['ice']['processor']['issue_100_workaround']    = false
 
-# S3 keys for accessing billing files
+# AWS access key id used for accessing AWS detailed billing files from S3.
 node.default['ice']['billing_aws_access_key_id']            = nil
+# AWS secret key used for accessing AWS detailed billing files from S3.
 node.default['ice']['billing_aws_secret_key']               = nil
 
+# Name of the S3 bucket containing the AWS detailed billing files.
 node.default['ice']['billing_s3_bucket_name']               = nil
+# Directory in the S3 billing bucket containing AWS detailed billing files.
 node.default['ice']['billing_s3_bucket_prefix']             = nil
 
-node.default['ice']['billing_payerAccountId']				= nil
-node.default['ice']['billing_accessRoleName']				= nil
-node.default['ice']['billing_accessExternalId']				= nil
+node.default['ice']['billing_payerAccountId']               = nil
+node.default['ice']['billing_accessRoleName']               = nil
+node.default['ice']['billing_accessExternalId']             = nil
 
+# Name of the S3 bucket that Ice uses for processed AWS detailed billing files.
+# This bucket is shared between the Ice processor and reader.
 node.default['ice']['work_s3_bucket_name']                  = nil
+
+# Directory in the S3 work bucket containing the processed AWS detailed billing files.
 node.default['ice']['work_s3_bucket_prefix']                = nil
 
+# Enables the Ice reader and installs the Nginx reverse proxy.
 node.default['ice']['reader']['enabled']                    = true
-node.default['ice']['reader']['local_dir']                   = '/var/ice_reader'
+# Local work directory for the Ice reader.
+node.default['ice']['reader']['local_dir']                  = '/var/ice_reader'
 
-node.default['ice']['start_millis']                          = 0
+# Specify the start time in milliseconds for the processor to start processing billing files.
+node.default['ice']['start_millis']                         = 0
 
+# Optional.  Hash mapping of AWS account names to account numbers.  This is used
+# within Ice to give accounts human readable names in the UI.
 node.default['ice']['accounts']                             = {}
 
+# Hash mapping accounts to IAM roles in order to allow reservations in that
+# account to be polled.
 node.default['ice']['reservation_capacity_poller']          = false
```

Comment:
Comment say hash map, but attribute appears to be a boolean value. If hash, should default to {}.